README.md

```diff
@@ -242,92 +238,61 @@ helm init
 
 ```bash
 helm repo add alfresco-incubator https://kubernetes-charts.alfresco.com/incubator
+helm repo add alfresco-stable https://kubernetes-charts.alfresco.com/stable
 ```
 
 ### 8. Add Local DNS
 
-Add Local DNS Entry for Host Machine (needed for JWT issuer matching). Be sure to specify an active network interface.  It is not always `en0` as illustrated.  You can use the command `ifconfig -a` to find an active interface.
+We will be forming a local dns with the use of nip.io. All you have to do it get your ip using the following command.
```

Comment:
`nip.io` is fine for local, experimental deployments. I could see enterprise customers having issues with depending on free 3rd party service.

Reply:
Yes, exactly, that's why did is just for local deployment with docker-for-desktop. Could also be xip.io.